### ISSUE TITLE
Home: auto-update

### ISSUE THREAD

**Issue by darosior** [revault/revault-gui#198]
Instead of having to click on the menu to refresh, it would be neat to have the interface updating itself automatically at a fixed interval.

**Comment by edouardparis** [comment]
Yes, the difficulty would be to reload but keep the scroll state and the focused objects
